js/widgets/fixedToolbar.js

```diff
@@ -72,15 +76,17 @@ define( [ "jquery", "../widget", "../core", "../animationComplete", "../navigati
 		},
 
 		_addTransitionClass: function() {
-			var tclass = this.options.transition;
+			var transitionClass = this.options.transition;
 
-			if ( tclass && tclass !== "none" ) {
+			if ( transitionClass && transitionClass !== "none" ) {
 				// use appropriate slide for header or footer
-				if ( tclass === "slide" ) {
-					tclass = this.element.hasClass( "ui-header" ) ? "slidedown" : "slideup";
+				if ( transitionClass === "slide" ) {
+					transitionClass = this.element.hasClass( "ui-toolbar-header" ) ? 
```

Comment:
I know this is not new but we should not be checking for a class here we know this already it should be checking `this.role`